Symfony: HTTP client tracing (#5845)
getsentry/sentry-docs: src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx

sentry:
  tracing:
    dbal: # DB queries
      enabled: false
```

> ### Twig

Reviewer: ```suggestion
## Twig
```

sentry:
  tracing:
    twig: # templating engine
      enabled: false
```

> ### Cache

Reviewer: ```suggestion
## Cache
```

sentry:
  tracing:
    cache: # cache pools
      enabled: false
```

### HTTP client

This integration adds support for distributed tracing of the Symfony HTTP client.

It is enabled by default. To disable it, set `enabled` to `false`.

```yaml {filename:config/packages/sentry.yaml}
sentry:
  tracing:
    http_client: # Symfony HTTP client
      enabled: false
```

> As the HTTP client tracing is based on Symfony's [TraceableHttpClient](https://github.com/symfony/symfony/blob/558fec00b0fa92dc1499434355e5a35de162bbe7/src/Symfony/Component/HttpClient/TraceableHttpClient.php) there are a few things to consider:

Reviewer: ```suggestion
As the HTTP client tracing is based on Symfony's [TraceableHttpClient](https://github.com/symfony/symfony/blob/558fec00b0fa92dc1499434355e5a35de162bbe7/src/Symfony/Component/HttpClient/TraceableHttpClient.php), you must consider the following:
```

> - To get correct span timings, do not assign the response of the request to a variable if you do not intend to use it.

Reviewer: ```suggestion
- To get correct span timings, don't assign the response of the request to a variable if you don't intend to use it.
```
